CodeStarterCamp_Week3/main.swift

```diff
@@ -1,12 +1,7 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week3
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
+let misterLee: Person = Person(name: "미스터 이", money: 5000)
+let missKim: Person = Person(name: "미스 김", money: 10000)
+let yagomBucks: CoffeeShop = CoffeeShop(sales: 500000, barista: misterLee)
```

Comment:
제가 드렸던 [이 코멘트](https://github.com/yagom-academy/swift-starter-week3/pull/81#discussion_r861843158) 내용이 전부 반영되지 않은 것 같습니다.

타입의 인스턴스를 받는 상수를 선언할 때, `Type annotation`을 생략하는 게 깔끔합니다. 👍🏻 
왜냐면, 가까운 거리에 중복 정보가 붙어있기 때문입니다.

```suggestion
let misterLee = Person(name: "미스터 이", money: 5000)
let missKim = Person(name: "미스 김", money: 10000)
let yagomBucks = CoffeeShop(sales: 500000, barista: misterLee)
```